docs/user-guide/security-features.md

```diff
@@ -1,20 +1,112 @@
 # Security Features
 
-{% hint style="info" %}
-### Prerequisite
+## Introduction
 
-Install any one of the following integrations for scanning vulnerabilities:
-* [Clair](../user-guide/integrations/clair.md)
-* Trivy
-{% endhint %}
+Devtron provides [DevSecOps](https://devtron.ai/product/devsecops) capabilities across your software development life cycle for both: the default CI/CD solution by Devtron as well as your existing CI/CD Tools.
+
+One of the key components of DevSecOps is the detection of security risks. Currently, Devtron supports the following types of scanning:
+
+* Image Scan
+* Code Scan (not available for Helm apps) 
```

Comment:
It's not like not available for helm apps, helm apps doesn't have codes attached with the chart to be scanned. This seems like we haven't supported it

Reply:
Fixed, removed the phrase altogether